prime-router/src/main/kotlin/LivdTableDownload.kt

```diff
@@ -0,0 +1,98 @@
+package gov.cdc.prime.router
```

Comment:
Move this file to at least the CLI folder. This file is small enough that you could also include this code with the LivdTableDownload class.

Reply:
Done.  Thanks